src/main/java/io/cloudslang/content/couchbase/actions/buckets/DeleteBucket.java

```diff
@@ -180,4 +208,11 @@
             return getFailureResultsMap(exception);
         }
     }
+
+    public static void main(String[] args) {
```

Comment:
hmm.. should the main methods be merged into master?

Reply:
:)